docs/index.adoc

```diff
@@ -150,31 +150,53 @@ It *must be unique* for your application.
 
 NOTE: See <<Feature Permissions>> for details on the various states a feature can have.
 
-=== Split targeting strategies and targeting rules
+=== Strategy Splits and general targeting rules
 
-Split targeting provides an ability to rollout features to a limited audience based on targeting rules, for example if you want to turn your feature on 
-for a user only on a mobile device or only with a specific email or userId. Apart from the targeting rules you can also split by percentage, or mix attributes with percentages.
+We refer to a feature flag in an as a Rollout Strategy. Each Rollout Strategy can have _multiple_
+split targeting strategies, i.e. a feature flag in each environment has one default value and zero or more splits.
 
-*How many split targeting strategies can a feature flag have?*
+A split consists of one or more rules. That rule can consist of any combination of matching criteria (such as if you want 
+to turn your feature on for a user only on a mobile device or only with a specific email or 
+userIds). 
 
-There can be *multiple split targeting strategies* assigned to rollout a feature flag. Imagine you have a feature flag of type string which controlls a "submit button color" that can be in multiple states, e.g green, blue, red etc. With rollout strategies, you can serve a `green` value to users on `iOS` devices, `blue` value to users whose emails ending with `gmail.com` and `red` value to users who are in `New Zealand or United States or United Kingdom`.   
+==== Each rule is an AND condition
```

Comment:
I would rename to "Split targeting rules". But there needs to be a section before that explains rules types - which I added in "What Rule types can I use....?"